src/nix/eval.md

```diff
@@ -50,8 +50,7 @@ R""(
 
 # Description
 
-This command evaluates the given Nix expression and prints the
-result on standard output.
+This command eagerly evaluates the given Nix expression and prints the result on standard output.
```

Comment:
Evaluation is never eager or strict. It always happens the same way.
The difference is in how evaluation is used; not intrinsic to evaluation itself.

```suggestion
This command also evaluates any nested attribute values and list items, and prints the result on standard output.
```